CONTRIBUTING.md

```diff
@@ -146,6 +146,10 @@ If you only want subset of these actions, you can use:
 
 - `npm run watch:build` to start a watcher task that **re-compiles Vega-Lite** when `.ts` files related to VL change.
 
+#### Fast iteration testing
```

Comment:
Well, I'm using it and it's sooooo fast. I can revise the text to say this is only for dev but you should run full tests before pushing.